packages/cfpb-core/src/utilities.less

```diff
@@ -13,6 +13,18 @@
   }
 }
 
+//
+// To hide an element when JS is on
+// And show it when JS is off
+//
+
+.js-is-hidden {
```

Comment:
I'm sorry I know it's more work, but could we update this to `u-js-is-hidden` to match the other utilities? Looks like it's only used in RAFT and CCDB, so these two references would just need an update:

- https://github.com/cfpb/consumerfinance.gov/blob/2c95562349cb5bf8273b1eda2229c19ba2116d9c/cfgov/jinja2/ccdb-complaint/ccdb-search.html#L22
- https://github.com/cfpb/consumerfinance.gov/blob/2c95562349cb5bf8273b1eda2229c19ba2116d9c/cfgov/v1/jinja2/v1/includes/blocks/raf_tool.html#L41

Reply:
suuuuuure. It's not more work in this PR, just in the one that remove this class from cf.gov ^_^